pandas/tests/groupby/conftest.py

```diff
@@ -76,3 +76,10 @@ def three_group():
                       'D': np.random.randn(11),
                       'E': np.random.randn(11),
                       'F': np.random.randn(11)})
+
+
+AGG_FUNCS = ['sum', 'prod', 'min', 'max', 'mean', 'median', 'var', 'first',
```

Comment:
While not specific to this PR I figured this was a good time to introduce this and increase / standardize coverage